src/index.js

```diff
@@ -1290,7 +1298,7 @@ export class ULabel {
                     $("#" + this.config["imwrap_id"]).prepend(this.filter_distance_overlay.get_canvas())
                     
                     // Filter the points with an override
-                    filter_points_distance_from_line(this, null, {
+                    filter_points_distance_from_line(this, true, null, {
```

Comment:
So this is `create_overlays`, which is called by `init`. It looks like `set_annotations` does call `update_filter_distance`, and should be calling this similarly, as long as the toolbox properly includes the `FilterDistanceItem` (is the toolbox loaded at that point?).

Are there any ways to perform an "in-place" reload that could end up not properly calling this?

Might be related to the QASM reloading issue (that doesn't appear to be present in `v0.11.0`).

Reply:
I haven't pushed the `set_annotations` fix to the qasm site yet, i suspect that was the issue.

I am tracking down a bug right now that I think has to do with this `create_overlays` call. To reproduce:

1) Open a fresh instance of the `0.12.0` site
2) Enable both `Show Filter Range` and `Filter During Move` (to make the bug easier to detect)
3) Move a row. Some points will stay visible when they should be deprecated.

But if you follow the same steps and instead first delete/create a row (operations that force a recalc of all distances) before moving a row, the bug disappears

Reply:
in local testing i thought this might be due to `move/edit/delete` logic sometime creating a new `annotation_id`, which was preventing the `polyline_id` from matching correctly as so points weren't updating their distances. But the bug still seems to be present on the site so 🤷

Reply:
I see this on the demo site, but not when running `row-filtering-example` on the local demo.

Reply:
jk looks like its fixed now. site just hadnt updated with my changes

Reply:
Yup, I agree. Prior to 1046840 I could reproduce, and now it appears to be gone.

Reply:
In-place reloading on QASM also appears fixed